mx.c

```diff
@@ -1400,9 +1398,9 @@ int mx_path_canon(struct Buffer *buf, const char *folder, enum MailboxType *type
   for (size_t i = 0; i < 3; i++)
   {
     /* Look for !! ! - < > or ^ followed by / or NUL */
-    if ((buf_at(buf,0) == '!') && (buf_at(buf,1) == '!'))
+    if ((buf_at(buf, 0) == '!') && (buf_at(buf, 1) == '!'))
```

Comment:
[clang-format](https://neomutt.org/dev/analysis/clang-format)
same for everything below